### DISCUSSION TITLE
transfer learning with DeePMD

### DISCUSSION THREAD

**Discussion by plumbum082** [deepmodeling/deepmd-kit#2160] [Q&A] [answered]
Hi~ I see [here](https://tutorials.deepmodeling.com/en/latest/CaseStudies/Transfer-learning/Transfer-learning.html) DP-GEN can do transfer learning using comand "simplify", however, I did not see similar comands in DeePMD. Now I have had a trained DeePMD model  and I want to do transfer learning with another dataset using DeePMD based on this model, I would like to know how could I implement transfer learning with DeePMD. @njzjz  Could you please help me? I will be appreciated!

**Comment by njzjz** [comment, marked as answer]
To train a model based on an existing model, use `dp train -f some_model.pb`. See [the documentation for `dp train`](https://docs.deepmodeling.com/projects/deepmd/en/v2.1.5/cli.html#train).

**Reply by plumbum082** [reply]
Okay, I will try. Thanks a lot for your help!
